lib/mapper.js

```diff
@@ -72,13 +72,21 @@ exports.track = function(track) {
       }
     });
   } else {
+    if (typeof track.proxy('context.traits.email') !== 'undefined'){
```

Comment:
Note: pretty new to the Facade and the layout of this code, so this may be a very rudimentary question!